services/gitdiff/gitdiff.go

```diff
@@ -739,6 +741,20 @@ parsingLoop:
 		}
 
 		curFile = createDiffFile(diff, line)
+		if skipping {
+			if curFile.Name != skipTo {
```

Comment:
If `--skip-to` was used, this should technically never evaluate to true?

Reply:
It doesn't matter. I can change it so that it skip-to is used skipping is set to false but the code works as is.